src/Mesh/babylon.mesh.vertexData.ts

```diff
@@ -1401,10 +1401,10 @@
          * Creates the VertexData of the TiledGround.  
          */
         public static CreateTiledGround(options: { xmin: number, zmin: number, xmax: number, zmax: number, subdivisions?: { w: number; h: number; }, precision?: { w: number; h: number; } }): VertexData {
-            var xmin = options.xmin || -1.0;
-            var zmin = options.zmin || -1.0;
-            var xmax = options.xmax || 1.0;
-            var zmax = options.zmax || 1.0;
+            var xmin = typeof options.xmin === 'number' ? options.xmin : -1.0;

```

Comment:
I would prefer:
var xmin = (options.xmin !== undefined && options.xmin !== null) ? options.xmin : -1.0;

Reply:
I'll do it :)

Reply:
Thanks man, been busy